services/communitytokensv2/api.go

```diff
@@ -483,13 +121,9 @@ func (api *API) SafeGetOwnerTokenAddress(ctx context.Context, chainID uint64, co
 	return api.s.SafeGetOwnerTokenAddress(ctx, chainID, communityID)
 }
 
-func (api *API) SetSignerPubKey(ctx context.Context, chainID uint64, contractAddress string, txArgs wallettypes.SendTxArgs, password string, newSignerPubKey string) (string, error) {
```

Comment:
I see this disappeared, was it not used in the client?

Reply:
@dlipicar correct, it's not used as an explicit API endpoint but goes over the router now.